go/vt/tabletserver/query_executor.go

```diff
@@ -49,6 +49,18 @@ func (qre *QueryExecutor) Execute() (reply *mproto.QueryResult, err error) {
 	defer func(start time.Time) {
 		duration := time.Now().Sub(start)
 		qre.qe.queryServiceStats.QueryStats.Add(planName, duration)
+
+		username := "unknown"
```

Comment:
"var username string" should be better since you don't use the constant "unknown" in your code